test/functional/feature_versionbits_warning.py

```diff
@@ -1,110 +0,0 @@
-#!/usr/bin/env python3
-# Copyright (c) 2016-2018 The Bitcoin Core developers
-# Distributed under the MIT software license, see the accompanying
-# file COPYING or http://www.opensource.org/licenses/mit-license.php.
-"""Test version bits warning system.
-
-Generate chains with block versions that appear to be signalling unknown
-soft-forks, and test that warning alerts are generated.
-"""
-import os
-import re
-
-from test_framework.blocktools import create_block, create_coinbase
-from test_framework.messages import msg_block
-from test_framework.mininode import P2PInterface, mininode_lock
-from test_framework.test_framework import BitcoinTestFramework
-from test_framework.util import wait_until
-
-VB_PERIOD = 144           # versionbits period length for regtest
-VB_THRESHOLD = 108        # versionbits activation threshold for regtest
-VB_TOP_BITS = 0x20000000
-VB_UNKNOWN_BIT = 27       # Choose a bit unassigned to any deployment
-VB_UNKNOWN_VERSION = VB_TOP_BITS | (1 << VB_UNKNOWN_BIT)
-
-WARN_UNKNOWN_RULES_MINED = "Unknown block versions being mined! It's possible unknown rules are in effect"
-WARN_UNKNOWN_RULES_ACTIVE = "unknown new rules activated (versionbit {})".format(VB_UNKNOWN_BIT)
```

Comment:
Why is this test removed?

this warning is still in the code and is still triggered in the gui, rpc and alertnotify

Reply:
Oh oops, I missed that as well in my review, that should probably stay.

Reply:
It seemed like this test was failing locally, will take a look.